### PR TITLE
Android course topics

### PR DESCRIPTION
**Pull request by bilalawan321**
###Description
- [LEARNER-7992](https://openedx.atlassian.net/browse/LEARNER-7992)

Followings features/stories will be implemented:
Verify that Course Topics screen will show following Header contents,
Back icon
Specific "<Topic name>" as Title
Verify that user should be able to go back by clicking Back icon
Verify that user should be able to view these on Every Topic in the list:
Topic name
Topic icon
download icon to video (if available)
Verify that on Clicking any topic Specific resource screen should be loaded successfully

### CODE REVIEW

**Comment by bilalawan321** [comment]
@farhan-arshad-dev  can you please review this PR ?

File: tests/android/tests/test_android_course_subsection.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+        topic_name = android_course_section_page.get_course_row_header().text
+        android_course_section_page.get_course_row_header().click()
+        assert android_course_dashboard_page.get_navigation_icon().get_attribute('content-desc') \
+            == strings.COURSE_DASHBOARD_NAVIGATION_ICON
+        android_course_dashboard_page.get_navigation_icon().click()
+        assert android_course_section_page.on_screen() == global_contents.COURSE_DASHBOARD_ACTIVITY_NAME
+
+        android_course_section_page.get_course_row_header().click()
+        if topic_name:
```
Comment (farhan-arshad-dev) [inline tests/android/tests/test_android_course_subsection.py:71]: deserve doc-string, like verify screen title.

**Review by farhan-arshad-dev — CHANGES_REQUESTED** [review]

File: tests/common/strings.py [hunk 2]
```diff
@@ -165,3 +165,7 @@
 COURSE_DASHBOARD_ANNOUNCEMENT_TITLE = 'Announcements'
 COURSE_DASHBOARD_ANNOUNCEMENT_ROW = 'Keep up with the latest news'
 COURSE_DASHBOARD_NAVIGATION_ICON = 'Navigate up'
+
+# COURSE SUBSECTION SCREEN
+COURSE_SUBSECTION_CONTENT_ROW_TEXT = 'Study Guide'
+COURSE_SUBSECTION_VIDEO_ROW_TEXT = 'Lecture Module 1: Introductions'
```
[the remark this answers is not in the document]
Comment (bilalawan321) [reply tests/common/strings.py:171]: fixed

File: tests/common/globals.py [hunk 3]
```diff
@@ -70,7 +70,7 @@ def __init__(self, project_log):
         self.fourteenth_existence = 13
         self.fifteenth_existence = 14
         self.sixteenth_existence = 15
-        self.jenkins = True
+        self.jenkins = False
```
[the remark this answers is not in the document]
Comment (bilalawan321) [reply tests/common/globals.py:73]: pushed it mistakenly. 
fixed

File: tests/android/pages/android_elements.py [hunk 4]
```diff
@@ -134,7 +134,13 @@
 course_layout = 'android.widget.LinearLayout'
 course_dashboard_course_image = 'org.edx.mobile:id/header_image_view'
 course_dashboard_course_name = 'org.edx.mobile:id/course_detail_name'
+course_dashboard_course_title = 'org.edx.mobile:id/course_detail_name'
```
[the remark this answers is not in the document]
Comment (bilalawan321) [reply tests/android/pages/android_elements.py:137]: fixed

File: tests/android/tests/test_android_course_subsection.py (hunk 1 above)
Comment (bilalawan321) [reply tests/android/tests/test_android_course_subsection.py:71]: fixed

**Review by farhan-arshad-dev — APPROVED** [review]

**Pull request merged**